ext/openssl/openssl.stub.php

```diff
@@ -105,39 +105,39 @@ function openssl_pkey_get_details(OpenSSLAsymmetricKey $key): array|false {}
 
 function openssl_pbkdf2(string $passphrase, string $salt, int $key_length, int $iterations, string $digest_algorithm = "sha1"): string|false {}
 
-function openssl_pkcs7_verify(string $filename, int $flags, ?string $output_filename = null, array $ca_info = [], ?string $untrusted_certificates_filename = null, ?string $content = null, ?string $pk7_filename = null): bool|int {}
+function openssl_pkcs7_verify(string $filename, int $flags, ?string $signers_certificates_filename = null, array $ca_info = [], ?string $untrusted_certificates_filename = null, ?string $content = null, ?string $pkcs7_output_filename = null): bool|int {}
```

Comment:
Just wondering what's the actual reason to use `array $ca_info = []`? Elsewhere it's for example `?array $header`... Is there any difference?

Reply:
As far as I can see, where we have the `$ca_info` parameter name, `php_openssl_setup_verify` is always used, while where we have the `$header` name, `BIO_printf` is used in a "foreach".

And the documentation says `headers is an array of headers that will be prepended to the data after it has been encrypted` (https://www.php.net/manual/en/function.openssl-pkcs7-encrypt.php), so it seems to me that they are distinct from each other.